src/component/meter/index.js

```diff
@@ -17,10 +17,10 @@ class Meter extends Component {
     const percentages = resultsArray.map(result => result / divisor);
 
     const svgProperties = [ 
-      { width: percentages[0], color: 'blue', star: '★' },
-      { width: percentages[1], color: 'green', star: '★★' },
-      { width: percentages[2], color: 'red', star: '★★★' },
-      { width: percentages[3], color: 'yellow', star: '★★★★' },
+      { width: percentages[0], color: '#F5839B', star: '★' },
```

Comment:
These colors need to be different. The green is too bright and the pink and magenta/purple are too close in color with each other. I prefer the old colors over this. Let's find a better color theme.